config.py

```diff
@@ -2,23 +2,36 @@
 
 SEED: int = 30
 BASE_URL: str = "https://cluster.klima.uni-bremen.de/~oggm/gdirs/oggm_v1.6/L3-L5_files/2023.1/elev_bands/W5E5_w_data/"
-META_DATA: List[str] = [
-    "RGIId", "POINT_ID", "ID", "N_MONTHS", "MONTHS", "PERIOD"
-]
-
-NOT_METADATA_NOT_FEATURES: List[str] = ["POINT_BALANCE", "YEAR", "POINT_LAT", "POINT_LON", 'ALTITUDE_CLIMATE']
 
 TEST_SIZE: float = 0.3
 N_SPLITS: int = 5
+
+# Default:
+META_DATA: List[str] = [
+    "RGIId", "POINT_ID", "ID", "N_MONTHS", "MONTHS"
+]
+NOT_METADATA_NOT_FEATURES: List[str] = [
+    "POINT_BALANCE", "YEAR", "POINT_LAT", "POINT_LON", 'ALTITUDE_CLIMATE'
+]
 NUM_JOBS: int = -1
+LOSS:str = 'MSE'
+
+# For CH
```

Comment:
For now ok like this (commented), but for the future perhaps better if we have config-dependent files for each of the regions.